metatron-app/metatron-discovery
# #3544 Added member sign-up to external login page and added organization input function when registering as a member

### Description
Added member sign-up to external login page and added organization input function when registering as a member

**Related Issue** : #3544 


### How Has This Been Tested?
1. Check if there is a sign up button on the external login page.
2. Make sure there is an Organization field on the signup page.
3. Check if the organization code is checked when entering the organization.
4. Check if membership registration is successful.

#### Need additional checks?

### Types of changes



### Checklist:


### Additional Context
